tools/codegen/gen-config/example_config.yaml

```diff
@@ -0,0 +1,35 @@
+resources:
+  push_based_log_export:
+    read:
+      path: /api/atlas/v2/groups/{groupId}/pushBasedLogExport
+      method: GET
+    create:
+      path: /api/atlas/v2/groups/{groupId}/pushBasedLogExport
+      method: POST
+    schema:
+      aliases:
+        group_id: project_id
+      
+      ignores: ["links"]
+      
+      overrides:
+        project_id:
+          description: "Unique 24-hexadecimal digit string that identifies your project."
+          plan_modifiers: [{
+              imports: [ "github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier" ],
+              definition: "stringplanmodifier.RequiresReplace()"
+          }]
+          validators: [{
+            imports: [
+            "github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator",
+            "github.com/hashicorp/terraform-plugin-framework/path"
+            ],
+            definition: "stringvalidator.ConflictsWith(path.MatchRoot(\"name\"))"
+          }]
+        
+        prefix_path:
+          computability:
+            optional: true
+            computed: true
+      
+      timeouts: ["create", "read", "delete"]
```

Comment:
will probably delete this file, only for reference.
Note: changed timeouts here to be under schema options, @AgustinBettati lmk if you prefer as in the example in the spec

Reply:
looks good as well, main aspect of placing within `additions` is to make clear is is adding a new attribute which does not come from the api spec.

Reply:
hmmm may be it could use a better naming in that case, `additions` might be too generic

Reply:
either work for me at this time, it can be moved to more nested level in the future if more appropriate